table/shell/src/main/java/alluxio/cli/table/command/TransformTableCommand.java

```diff
@@ -74,7 +74,9 @@ public String getDescription() {
         + "\n\n"
         + "Files are coalesced and converted to parquet format."
         + "\n\n"
-        + "Definition is in format like 'write(hive).option(hive.num.files, 100)'.";
+        + "Definition is in format like 'write(hive).option(hive.num.files, 100)',"
+        + "currently, the only modification you can do to the example definition is "
+        + "the number of files in the option."
```

Comment:
I think this is better suited to live in the documentation page.  Maybe have a short reference section of some kind with all the available options - rather than putting this in the command output?

Reply:
I think since this will change in future release, it's better to let it be here so it's easier to update.

Reply:
All right - but if we start adding more I think it is better suited to live in the documentation, or have some kind of pretty-printed output that we can provide to users with the full list of options. Otherwise, no one will really know how to use it